luigi/parameter.py

```diff
@@ -995,3 +996,74 @@ def parse(self, s):
                 raise OSError("could not make parent directories for {s}".format(
                     s=s))
         return s
+
+
+class NumericalParameter(Parameter):
+    """
+    Parameter whose value is a number of the specified type, e.g. ``int`` or
+    ``float`` and in the range specified.
+
+    In the task definition, use
+
+    .. code-block:: python
+
+        class MyTask(luigi.Task):
+            my_param = luigi.NumericalParameter(
+                var_type=int, min_value=-3, max_value=7, left_op=operator.le, right_op=operator.lt)
```

Comment:
What about having two lines here instead, in particular I want to clarify what the default value of `left_op` and `right_op` is:

```python

my_param_1 = luigi.NumericalParameter(
    var_type=int, min_value=-3, max_value=7)  # -3 <= my_param_1 < 7
my_param_2 = luigi.NumericalParameter(
    var_type=int, min_value=-3, max_value=7, left_op=operator.lt, right_op=operator.le) # -3 < my_param_1 <= 7
```

------

Actually, isn't the default values for the operators quite unintuitive? I just would assume the default would be `[, )` or `lb <= x < ub` (I imagine `0 <= x < n` being the common usecase). But it looks like the default is the opposite right???